test/controllers/patients_controller_test.rb

```diff
@@ -123,4 +123,30 @@ class PatientsControllerTest < ActionController::TestCase
       assert_redirected_to root_path
     end
   end
+
+  #confirm get :data_entry returns a success code
+  describe `data_entry method` do
+    before do
```

Comment:
On this first test, we aren't doing anything with data, so you can cut this `before` block. You should keep it for the create method one though!